### PR TITLE
joconner adds hometown Ventura, CA

### PR DESCRIPTION
**Pull request by joconner**
Hey,

If you're in Ventura, CA, what did you visit? Any cool restaurants?

Do you have a favorite beach?

### CODE REVIEW

File: joconner/ventura.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Ventura, CA
+
+Ventura, CA is a beautiful city on the Pacific coastline.
```
Comment (rakeshnrao) [inline joconner/ventura.md]: Hi John, Nice meeting you virtually... Did not realize that you live in Ventura. Seems to be an awesome place.

**Comment by ccapurroroy** [comment]
I like Sharkless Beach

**Comment by joconner** [comment]
> I like Sharkless Beach

@ccapurroroy You'll love that beach, but the sharks make things more interesting.

**Comment by githubteacher** [comment]
@joconner :shipit: looks ✨  to me!

**Pull request merged**